src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+const Button =  function(props){
```

Comment:
Good job in making smaller components, though keep in mind it is more consistent to use ES6 `const Button = (props) => {...}` to get used to it in the long run